### DISCUSSION TITLE
Feedback on CroftEngine in Linux

### DISCUSSION THREAD

**Discussion by Leebre** [stohrendorf/CroftEngine#258] [General]
Hi, I have compiled and run CroftEngine in Linux (Arch variant) on an x86 desktop PC and just wanted to give some feedback:

Overall, it seems like a good engine; however, I ran into some persistent issues with the game crashing when saving/loading and when Lara dies. I have submitted a bug report for these.

I have also noticed a couple of other (more minor) issues:

- fullscreen doesn't seem to work properly - I still see the top/bottom panels of my desktop environment
- the image in-game often seems to become a bit blurry, which seems to relate to certain positions of the camera (?)

I will post issue reports for these as well, and provide screenshots.

In addition, a couple of suggestions:

- it would be great to have a brightness slider in the settings. The graphics seem very bright, which doesn't seem to fit the 'tomb raider' theme very well
- in my opinion, it would be better to make the default controls the same as those of the original game
- is there a level-skip cheat available? There are a couple of particular levels that I would like to try the engine with

Otherwise, again it's a really solid effort and it's great to see a good, open-source TR engine that works natively in Linux  :-)

**Comment by stohrendorf** [comment]
The keyboard controls are laid out to be able to play with one hand only. It's debatable, though, and as long as I don't hear strong points for the original layout, my personal preference wins(TM).

The original PC cheats are available. Step forward step backwards, then one and a half full turns on the spot. Jumping backwards will give you weapons and ammo, jumping forward will skip the level.

A brightness/contrast slider is already lingering in my mind, but I simply forgot to write tickets for these.

**Reply by Leebre** [reply]
Thanks for your quick reply! I understand the keyboard default is a matter of preference and that's fair enough. Perhaps you could put me as a +1 for the original layout, in case other people make a similar request?

Thanks also for the tip on the level skip cheat and for considering the brightness/contrast setting as well. I think those would be really useful  :-)
